docs/production-install.rst

```diff
@@ -105,7 +97,7 @@ for more information on this file.
 
 You'll need to restart postgresql if the configuration was updated:
 ::
-  sudo service postgresql-9.3 restart
+  sudo systemctl restart postgresql-9.3
 
 Disable SELinux
```

Comment:
@phrawzty I agree - I think the main thing we need to do is stop dumping everything into ```/data/socorro```, which we'll be clear to do after we decom PHX (bug 1145396)